app/src/main/java/net/squanchy/tweets/view/TweetUrlSpanFactory.kt

```diff
@@ -0,0 +1,24 @@
+package net.squanchy.tweets.view
+
+import android.content.Context
+import android.content.res.Resources
+import android.support.annotation.AttrRes
+import android.support.annotation.ColorInt
+import android.util.TypedValue
+
+import net.squanchy.R
+
+class TweetUrlSpanFactory(private val context: Context) {
```

Comment:
I'm quite sure we can convert this to a single top-level function (and a private one)

Reply:
Not really sure about this, because then we would have to pass the context around to the service or wrap it anyway so I guess we can just keep this at the moment